build/bootstrap.proj

```diff
@@ -14,8 +14,8 @@
         <PackageDownload Include="Lucene.Net" version="[3.0.3]" />
         <PackageDownload Include="MicroBuild.Core" version="[0.2.0]" />
         <PackageDownload Include="Microsoft.Build" version="[15.1.262-preview5]" />
-        <PackageDownload Include="Microsoft.DotNet.Build.Tasks.Feed" version="[5.0.0-beta.20224.2]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
-        <PackageDownload Include="Microsoft.DotNet.Maestro.Tasks" version="[1.1.0-beta.20352.2]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
+        <PackageDownload Include="Microsoft.DotNet.Build.Tasks.Feed" version="[6.0.0-beta.20508.7]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
```

Comment:
Where were these coming from before? 

Specifically, I see that you've added a new feed, so I'm wondering if we have an extra feed right now.

Reply:
Yes, new feed is going to be used..

Reply:
My question was whether https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json was added for anything more than just this package, and whether we can remove, but quick research says no. Other packages are coming from there.